examples/with-react-native-web/packages/tsconfig/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "tsconfig",
+  "version": "0.0.0",
+  "license": "MIT",
+  "main": "index.js",
+  "files": [
```

Comment:
I'd prefer dropping this as they're not really being published; it's just going to be a troll if somebody ever does do artifactory.

Reply:
👍🏼 done.